administrator/language/de-DE/com_media.ini

```
@@ -25,6 +25,11 @@ COM_MEDIA_DELETE_SUCCESS="Datei gelöscht."
 COM_MEDIA_DROP_FILE="Datei per Drag-and-Drop hochladen"
 COM_MEDIA_EDIT="Media bearbeiten"
 COM_MEDIA_ERROR="Es ist ein Fehler aufgetreten."
+COM_MEDIA_ERROR_ACCOUNT_NOT_FOUND="Das Konto wurde nicht gefunden"
+COM_MEDIA_ERROR_ACCOUNT_NOT_SET="Konto wurde nicht eingerichtet."
+COM_MEDIA_ERROR_MEDIA_PROVIDER_NOT_FOUND="Medien-Provider nicht gefunden."
+COM_MEDIA_ERROR_NO_ADAPTER_FOUND="Kein Adapter gefunden."
+COM_MEDIA_ERROR_NO_CONTENT_AVAILABLE="Kein Inhalt verfügbar!"
```

```suggestion
COM_MEDIA_ERROR_NO_CONTENT_AVAILABLE="Kein Inhalt verfügbar."
```

Oder sollen wir hier ! behalten?